ui/wdio.conf.js

```diff
@@ -1,280 +1,276 @@
-     */
-    // beforeCommand: function (commandName, args) {
-    // },
-    /**
-     * Hook that gets executed before the suite starts
-     * @param {Object} suite suite details
-     */
-    // beforeSuite: function (suite) {
-    // },
-    /**
-     * Function to be executed before a test (in Mocha/Jasmine) starts.
-     */
-    // beforeTest: function (test, context) {
-    // },
-    /**
-     * Hook that gets executed _before_ a hook within the suite starts (e.g. runs before calling
-     * beforeEach in Mocha)
-     */
-    // beforeHook: function (test, context) {
-    // },
-    /**
-     * Hook that gets executed _after_ a hook within the suite starts (e.g. runs after calling
-     * afterEach in Mocha)
-     */
-    // afterHook: function (test, context, { error, result, duration, passed, retries }) {
-    // },
-    /**
-     * Function to be executed after a test (in Mocha/Jasmine).
-     */
-    // afterTest: function(test, context, { error, result, duration, passed, retries }) {
-    // },
+  ],
+  //
+  // ===================
+  // Test Configurations
+  // ===================
+  // Define all options that are relevant for the WebdriverIO instance here
+  //
+  // Level of logging verbosity: trace | debug | info | warn | error | silent
+  logLevel: 'info',
+  //
+  // Set specific log levels per logger
+  // loggers:
+  // - webdriver, webdriverio
+  // - @wdio/browserstack-service, @wdio/devtools-service, @wdio/sauce-service
+  // - @wdio/mocha-framework, @wdio/jasmine-framework
+  // - @wdio/local-runner
+  // - @wdio/sumologic-reporter
+  // - @wdio/cli, @wdio/config, @wdio/utils
+  // Level of logging verbosity: trace | debug | info | warn | error | silent
+  // logLevels: {
+  //     webdriver: 'info',
+  //     '@wdio/appium-service': 'info'
+  // },
+  //
+  // If you only want to run your tests until a specific amount of tests have failed use
+  // bail (default is 0 - don't bail, run all tests).
+  bail: 0,
+  //
+  // Set a base URL in order to shorten url command calls. If your `url` parameter starts
+  // with `/`, the base url gets prepended, not including the path portion of your baseUrl.
+  // If your `url` parameter starts without a scheme or `/` (like `some/path`), the base url
+  // gets prepended directly.
+  baseUrl: `http://localhost:${process.env.PORT}`,
+  //
+  // Default timeout for all waitFor* commands.
+  waitforTimeout: 10000,
+  //
+  // Default timeout in milliseconds for request
+  // if browser driver or grid doesn't send response
+  connectionRetryTimeout: 120000,
+  //
+  // Default request retries count
+  connectionRetryCount: 3,
+  //
+  // Test runner services
+  // Services take over a specific job you don't want to take care of. They enhance
+  // your test setup with almost no effort. Unlike plugins, they don't add new
+  // commands. Instead, they hook themselves up into the test process.
+  services: [['selenium-standalone', { drivers: { chrome: '95.0.4638.69' } }]],
```

Comment:
Does this work with just the swap to selenium-standalone, but not pinning the version? Keeping the version fresh is probably preferable.

Reply:
I agree we want to be able to test against the edge of chrome, but I think we should be able to choose when we do that. At the moment integration tests will fail at the cadence of the chrome update cycle.  

I'm pinning the driver to the version we're setting in [package.json](https://github.com/nhsx/standards-registry/blob/STAN-174-search-squish/ui/package.json#L34) (95), and I think the service is a passthrough to [`wdio-chromedriver-service@7.2.2`](https://github.com/nhsx/standards-registry/blob/STAN-174-search-squish/ui/package.json#L37)

Reply:
How come it's changed from `chromedriver` to `selenium-standalone`?

Reply:
Ah I understand your question a bit better now. 
```suggestion
  services: [['selenium-standalone']],
```
Let's try it out!

Reply:
That works. The default configuration of selenium-standalone is to take whatever's on the host (as far as I can tell) https://webdriver.io/docs/selenium-standalone-service/#configuration. The default configuration for chromedriver is to wrap a static version https://webdriver.io/docs/wdio-chromedriver-service

Reply:
Probably want to swap out the chromedriver service for the selenium service in package.json